test/unit/specs/components/staking/__snapshots__/PageProposal.spec.js.snap

```diff
@@ -0,0 +1,279 @@
+              </div>
+            </div>
+            <div
+              class="tabs"
+            />
+          </div>
+        </div>
+      </div>
+      <menu
+        class="tm-page-header-menu"
+      >
+        <div>
+          <div>
+            <div
+              class="tm-tool-bar"
+            >
+              <div
+                class="tm-tool-bar-container"
+              >
+                <div
+                  class="main"
+                >
+                  <a
+                    class=""
+                    href="#/governance"
+                  >
+                    <i
+                      class="material-icons"
+                    >
+                      arrow_back
+                    </i>
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </menu>
+    </div>
+  </header>
+  <main
+    class="tm-page-main ps"
+  >
+    <div
+      class="validator-profile__header validator-profile__section proposal"
+    >
+      <div
+        class="column validator-profile__header__info"
+      >
+        <div
+          class="row validator-profile__header__name"
+        >
+          <div
+            class="top column"
+          >
+            <div
+              class="validator-profile__status-and-title"
+            >
+              <span
+                class="validator-profile__status green"
+              />
+              <div
+                class="validator-profile__header__name__title"
+              >
+                Proposal Title
+              </div>
+            </div>
+          </div>
+          <div
+            class="column validator-profile__header__actions"
+          >
+            <!---->
+            <!---->
+            <button
+              class="tm-btn"
+              disabled="disabled"
+            >
+              <span
+                class="tm-btn__container tm-btn--primary"
+              >
+                <!---->
+                <!---->
+                <span
+                  class="tm-btn__value"
+                >
+                  Deposit / Vote
+                </span>
+              </span>
+            </button>
+          </div>
+        </div>
+        <div
+          class="row description"
+        >
+          <p>
+            This text proposal (#1) was submitted at block #120 and voting started at block #135.
+          </p>
+        </div>
+        <div
+          class="row validator-profile__header__data votes"
+        >
+          <dl
+            class="colored_dl"
+          >
+            <dt>
+              Deposit
+            </dt>
+            <dd>
+              100 stake
+            </dd>
+          </dl>
+          <div
+            class="validator-profile__header__data__break"
+          />
+          <dl
+            class="colored_dl"
+          >
+            <dt>
+              Yes
+            </dt>
+            <dd>
+              500 / 0%
```

Comment:
0 % ?

Reply:
lol yeah, i guess it should say 0 instead of 0%